public/js/components/App.js

```diff
@@ -32,18 +33,16 @@ const App = React.createClass({
   },
 
   componentDidMount() {
-    window.addEventListener("keydown", this.toggleSourcesSearch, false);
+    this.shortcuts = new KeyShortcuts({ window });
+    this.shortcuts.on("Cmd+P", this.toggleSourcesSearch, { preventDefault: true });
   },
 
   componentWillUnmount() {
-    window.removeEventListener("keydown", this.toggleSourcesSearch, false);
+    this.shortcuts.off("Cmd+P", this.toggleSourcesSearch);
```

Comment:
You can do `CmdOrCtrl+P` which will handle control on windows/linux or command on osx